cmd/influxd/launcher/launcher.go

```diff
@@ -576,57 +577,62 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 		return err
 	}
 
-	var deleteService platform.DeleteService
-	var pointsWriter storage.PointsWriter
-	{
+	if m.testing {
+		// the testing engine will write/read into a temporary directory
+		engine := NewTemporaryEngine(m.StorageConfig, storage.WithRetentionEnforcer(bucketSvc))
```

Comment:
@e-dard I vaguely remember you telling me I should not have a retention enforcer... is that right?